nats-box.yaml

```diff
@@ -0,0 +1,45 @@
+package:
+  name: nats-box
+  version: 0.17.0
+  epoch: 0
+  description: NATS utilities toolbox.
+  dependencies:
+    runtime:
+      - bash
+      - ca-certificates
+      - curl
+      - figlet
+      - jq
+  copyright:
+    - license: Apache-2.0
+
+environment:
+  contents:
+    packages:
+      - busybox
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/nats-io/nats-box
+      tag: v${{package.version}}
+      expected-commit: b73821692fc596b210816de93c5011047298646b
+
+  - runs: |
+      mkdir -p ${{targets.destdir}}/etc/profile.d
```

Comment:
Should we also need to create additional dirs? `/nsc`, `/home/nats`, `/nsc/nkeys`, `/nsc/.config`

Reply:
I've created them at Image level since they need certain permissions as well